docs/SUMMARY.md

```diff
@@ -36,6 +36,8 @@
 ### 2.3 Customization 
 * [Customizing your cBioPortal Instance via portal.properties](Customizing-your-instance-of-cBioPortal.md)
 * [More portal.properties Settings](portal.properties-Reference.md)
+* [How does the page layout the charts](study-view/layout.md)
```

Comment:
@zhx828 maybe better:	`How does the study view organize the charts`

Reply:
layout is a noun, so either `lay out` as a verb but that meaning is slightly different